library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -128,6 +128,10 @@
     width: 26vw;
     height: auto;
   }
+
+  .portal-main-container {
+    display: block !important;
```

Comment:
is it possible to avoid using !important and rely on css specificity?

Reply:
I am still trying to find out other ways but unable to do so yet.

Reply:
if you can describe the intent or the problem, we can suggest a solution. is some other code setting portal-main-container to inline or inline-block? if yes, do we know why?

Reply:
It also _looks_ like this change was already committed as part of #12787? I wonder why it's not being flagged as a conflict.

Reply:
@ottenhoff I found an alternative bootstrap class "d-grid" to fix this issue. Please let me know if that works - the issue has been fixed. 

@jonespm I will fix that - it is supposed to be only a part of this commit.